debian/ralf.init.d

```diff
@@ -102,8 +102,18 @@ get_daemon_args()
           billing_realm_arg="--billing-realm=$home_domain"
         fi
 
+        # Default the diameter timeout to twice the target latency if not
+        # already overridden (rounding up).  Note that the former is expressed
+        # in milliseconds and the latter in microseconds, hence division by 500
+        # (i.e. multiplication by 2/1000).
+        if [ -z $diameter_timeout_ms ] && [ ! -z $target_latency_us ]
```

Comment:
I wonder if this makes more sense as a Utils function (I assume this is just copied from Homestead).